src/fides/api/ops/api/v1/endpoints/policy_endpoints.py

```diff
@@ -297,6 +366,82 @@ def delete_rule(
     rule.delete(db=db)
 
 
+def get_rule_target_or_error(
+    db: Session,
+    policy_key: FidesOpsKey,
+    rule_key: FidesOpsKey,
+    rule_target_key: FidesOpsKey,
+) -> RuleTarget:
+    """
+    Helper method to load Rule Target or throw a 404.
+    Also throws a 404 if a `Policy` or `Rule` with the given keys can't be found.
+    """
+    logger.info("Finding rule target with key '{}'", rule_target_key)
+    rule: Rule = get_rule_or_error(db, policy_key, rule_key)
+    rule_target = RuleTarget.filter(
+        db=db,
+        conditions=(
+            (RuleTarget.rule_id == rule.id) & (RuleTarget.key == rule_target_key)
+        ),
+    ).first()
+    if not rule_target:
+        raise HTTPException(
+            status_code=HTTP_404_NOT_FOUND,
+            detail=f"No Rule Target found for key {rule_target_key}.",
+        )
+
+    return rule_target
+
+
+@router.get(
+    urls.RULE_TARGET_LIST,
+    status_code=HTTP_200_OK,
+    response_model=Page[schemas.RuleTarget],
+    dependencies=[Security(verify_oauth_client, scopes=[scope_registry.RULE_READ])],
+)
+def get_rule_target_list(
+    *,
+    db: Session = Depends(deps.get_db),
+    policy_key: FidesOpsKey,
+    rule_key: FidesOpsKey,
+    params: Params = Depends(),
+) -> AbstractPage[RuleTarget]:
+    """
+    Return a paginated list of all `Rule Target` records associated with the given `Rule`
+    Throws a 404 if the given `Rule` or `Policy` can't be found.
+    """
+    logger.info(
+        "Finding all rule targets for rule {} with pagination params '{}'",
+        rule_key,
+        params,
+    )
+    rule = get_rule_or_error(db, policy_key, rule_key)
+    rule_targets = RuleTarget.filter(
+        db=db,
+        conditions=(RuleTarget.rule_id == rule.id),
+    ).order_by(RuleTarget.created_at.desc())
+    return paginate(rule_targets, params=params)
+
+
+@router.get(
+    urls.RULE_TARGET_DETAIL,
+    status_code=HTTP_200_OK,
+    response_model=schemas.RuleTarget,
+    dependencies=[Security(verify_oauth_client, scopes=[scope_registry.RULE_READ])],
+)
+def get_rule_target(
+    *,
+    policy_key: FidesOpsKey,
+    rule_key: FidesOpsKey,
+    rule_target_key: FidesOpsKey,
+    db: Session = Depends(deps.get_db),
+) -> schemas.RuleTarget:
```

Comment:
I think this return type should be the RuleTarget model, not the schema, because that's what this method literally returns.  It gets serialized with the RuleTarget schema due to the `response_model` declaration above, then hopefully you can get rid of `# type: ignore[return-value]`

Existing endpoints in this file may make this same mistake, but compare to the connection config endpoints for example.